src/components/crownConfig.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <div class="crown-config" :style="style" v-if="showCrown">
+    <sequence-flow-button
+      @click="addSequence()"
+      class="crown-config__icon"
+      :title="$t('Sequence Flow')"
+    />
+    <message-flow-button
+      @click="addMessageFlow()"
+      class="crown-config__icon"
+      :title="$t('Message Flow')"
+    />
+    <delete-button
+      @click="removeShape()"
+      class="crown-config__icon"
+      :title="$t('Delete')"
+    />
+  </div>
+</template>
+
+<script>
+import DeleteButton from '@/components/deleteButton';
+import MessageFlowButton from '@/components/messageFlowButton';
+import SequenceFlowButton from '@/components/sequenceFlowButton';
+import pull from 'lodash/pull';
+
+export default {
+  components: {
+    DeleteButton,
+    MessageFlowButton,
+    SequenceFlowButton,
+  },
+  props: [
+    'highlighted',
+    'paper',
+    'graph',
+    'shape',
+    'node',
+    'nodeRegistry',
+    'moddle',
+    'planeElements',
+    'processNode',
+    'collaboration',
+  ],
+  watch: {
+    highlighted() {
+      this.showCrown = this.highlighted;
+    },
+    shape() {
+      if (this.highlighted) {
+        this.showCrown = true;
+        this.configurePoolLane();
+        this.setUpCrownConfig();
```

Comment:
In the original crown config, these two functions are only called once on mount. And even though this is in a watcher, `shape` should never change, and that seems to be the case here, as this watcher seems to only be called once after dragging the shape onto the grid.

Maybe break this out into a separate function that is only called once in the mounted hook?

Reply:
I had issues where it was erroring due to shape not being there. Is there a nice way to call it once, knowing that shape is initially null?

Reply:
I guess I can try next tick